common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -88,10 +88,10 @@
                 </div>
             }
 
-            case Some(media@InlineYouTubeMediaAtom(_)) => {
+            case Some(media@InlineYouTubeMediaAtom(_)) if item.cardTypes.showYouTubeMediaAtomPlayer => {
```

Comment:
Yeah that's fine, that's probably how I'd do it but we can make that change in the following PR.